examples/ERC20_solidity_compatible/ERC20.v.py

```diff
@@ -0,0 +1,97 @@
+# Solidity-Compatible EIP20/ERC20 Token
+# Implements https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20-token-standard.md
+# Author: Phil Daian
+
+# The use of the num256 datatype as in this token is not
+# recommended, as it can pose security risks.
+
+# Events are not yet supported in Viper, so events are NOT
+# included in this token.  This makes this token incompatible
+# with some log-only clients.
+
+# This token is intended as a proof of concept towards
+# language interoperability and not for production use.
+
+# Events issued by the contract
+Transfer: __log__({_from: indexed(address), _to: indexed(address), _value: num256})
+Approval: __log__({_owner: indexed(address), _spender: indexed(address), _value: num256})
+
+balances: num256[address]
+allowances: (num256[address])[address]
+num_issued: num256
+
+# Utility functions for overflow checking
+@constant
+def is_overflow_add(a : num256, b : num256) -> bool:
+    result = num256_add(a, b)
+    return num256_lt(result, a)
+
+@constant
+def is_overflow_sub(a : num256, b : num256) -> bool:
+    return num256_lt(a, b)
+
```

Comment:
I think `is_overflow_add` and `is_overflow_sub` are unnecessary due to pr #385

Reply:
Waiting on #411